packages/cli/generation/ir-generator/src/converters/services/convertCodeSamples.ts

```diff
@@ -0,0 +1,52 @@
+import { ExampleCodeSample, SupportedSdkLanguage } from "@fern-api/ir-sdk";
+import { RawSchemas, visitExampleCodeSampleSchema } from "@fern-api/yaml-schema";
+import { FernFileContext } from "../../FernFileContext";
+
+export function convertCodeSample({
+    codeSample,
+    file
+}: {
+    codeSample: RawSchemas.ExampleCodeSampleSchema;
+    file: FernFileContext;
+}): ExampleCodeSample {
+    return visitExampleCodeSampleSchema<ExampleCodeSample>(codeSample, {
+        language: (languageScheme) =>
+            ExampleCodeSample.language({
+                name: languageScheme.name != null ? file.casingsGenerator.generateName(languageScheme.name) : undefined,
+                docs: languageScheme.docs,
+                language: languageScheme.language,
+                code: languageScheme.code,
+                install: languageScheme.install
+            }),
+        sdk: (sdkScheme) =>
+            ExampleCodeSample.sdk({
+                name: sdkScheme.name != null ? file.casingsGenerator.generateName(sdkScheme.name) : undefined,
+                docs: sdkScheme.docs,
+                sdk: removeSdkAlias(sdkScheme.sdk),
+                code: sdkScheme.code
+            })
+    });
+}
+
+function removeSdkAlias(sdk: RawSchemas.SupportedSdkLanguageSchema): SupportedSdkLanguage {
```

Comment:
Are we referencing these cases from somewhere? It might be helpful to include a link to whatever markdown and/or code samples document we're (presumably) referencing. Otherwise they feel arbitrary and it's difficult to know how to update in the future.

I know this is just copy/pasted from another file in this change, so can handle this later if we don't know the original source.